client/public/index.html

```diff
@@ -21,14 +21,29 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
 
-    <!-- Google Tag Manager -->
-    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-      })(window,document,'script','dataLayer','GTM-NMPT5JP');</script>
+    <!-- 
+      Google Tag Manager
+       
+      Note that we insert the GTM script tag from JS to ensure that
+      GTM only initializes itself once we've set the title tag.
+      For more details, see Page.tsx in this repository.
+    -->
+    <meta name="jf-gtm-id" content="GTM-NMPT5JP">
+    <script>
+      (function (window, dataLayer) {
+        window.dataLayer = window.dataLayer || [];
+        window.dataLayer.push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
+      })(window, document, "script", "dataLayer", "GTM-NMPT5JP");
```

Comment:
Actually, I think you can just remove the [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) wrapper entirely and just have the two lines of the function run as top-level code... that will also avoid the confusion of the fact that the function's arguments aren't used at all too!